server/src/main/java/org/elasticsearch/threadpool/FixedExecutorBuilder.java

```diff
@@ -87,45 +66,16 @@
      */
     public FixedExecutorBuilder(final Settings settings, final String name, final int size, final int queueSize, final String prefix,
                                 final boolean trackEWMA) {
-        this(settings, name, size, queueSize, prefix, trackEWMA, false);
-    }
-
-    /**
-     * Construct a fixed executor builder.
-     *
-     * @param settings   the node-level settings
-     * @param name       the name of the executor
-     * @param size       the fixed number of threads
-     * @param queueSize  the size of the backing queue, -1 for unbounded
-     * @param prefix     the prefix for the settings keys
-     * @param trackEWMA  whether to track the exponentially weighted moving average of the task execution time
-     * @param deprecated whether or not the thread pool is deprecated
-     */
-    public FixedExecutorBuilder(
-        final Settings settings,
-        final String name,
-        final int size,
-        final int queueSize,
-        final String prefix,
-        final boolean trackEWMA,
-        final boolean deprecated
-    ) {
         super(name);
         final String sizeKey = settingsKey(prefix, "size");
-        final Setting.Property[] properties;
-        if (deprecated) {
-            properties = new Setting.Property[]{Setting.Property.NodeScope, Setting.Property.Deprecated};
-        } else {
-            properties = new Setting.Property[]{Setting.Property.NodeScope};
-        }
         this.sizeSetting =
-            new Setting<>(
-                sizeKey,
-                s -> Integer.toString(size),
-                s -> Setting.parseInt(s, 1, applyHardSizeLimit(settings, name), sizeKey),
-                properties);
+                new Setting<>(
```

Comment:
NIT: is the indent change intended here?

Reply:
So, to deprecate the listener thread pool, I added some temporary infrastructure in this class (see #53266) which changed the indentation here because of my IDE having different settings than when I authored this code a long time ago. 🤷‍♀

Since after removing this setting this infrastructure isn't needed anymore, I simply reverted the file back to its initial state before #53266, including the original formatting.

Long live automatic code formatting!